Example/Emission.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think that's an IDE artifact that doesn't need to be checked in

Reply:
Yep, checked it out of this branch.